packages/Guard/src/index.js

```diff
@@ -0,0 +1,10 @@
+import Supervisor from "./GuardSupervisor";
+import Connector from "./GuardConnector";
+import Gatekeeper, { useGatekeeper } from "./GuardGatekeeper";
+
+export default {
+  Supervisor,
+  Connector,
+  Gatekeeper,
+  useGatekeeper,
```

Comment:
Do you think this would make sense to export as `useGuard`? (to use the name of the main component)

Reply:
Maaaaybe 🤔